Libraries/LibWeb/CSS/CountersSet.cpp

```diff
@@ -116,125 +116,6 @@ void CountersSet::append_copy(Counter const& counter)
     m_counters.append(counter);
 }
 
-// https://drafts.csswg.org/css-lists-3/#auto-numbering
-void resolve_counters(DOM::AbstractElement& element_reference)
```

Comment:
I disagree with this move, I think keeping the counters code in one place (where possible) is beneficial.